### DISCUSSION TITLE
Bug: User Profile Page Displaying Incorrectly

### DISCUSSION THREAD

**Discussion by yanyaoli** [community/community#67867] [Mobile] [answered]
The user profile page is experiencing display issues. Some of the information within the user profile extends beyond its designated space, causing overlapping with other profile labels. Even after multiple refresh attempts, this issue persists and can be consistently reproduced.

Sample image:
![image](https://github.com/community/community/assets/120553430/ceb818a7-8beb-491d-be51-13ac9c6fc96b)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.131.0 (110081196)
Device: iPhone Xʀ (iOS 16.3)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @ohnonoyo thanks for writing in. Seems like this is definitely a problem, I filed an issue with the team to get this fixed.
